### PR TITLE
Hotfix/contour filtering bounding box

### PR DESCRIPTION
**Pull request by OrangePear13**
Fixed small issue in the creation of BoundingBox objects from `vision/common/bounding_box.py`

### CODE REVIEW

File: vision/emergent_object/emergent_object_processing.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
+import numpy as np
```
Comment (fallscameron01) [inline vision/emergent_object/emergent_object_processing.py:1]: Was this file meant to be in this PR?

**Review by fallscameron01 — CHANGES_REQUESTED** [review]
Requesting clarification. I think you accidentally included your emergent object processing code in this PR.

File: vision/emergent_object/emergent_object_processing.py (hunk 1 above)
Comment (eblake003) [reply vision/emergent_object/emergent_object_processing.py:1]: I made the same mistake because I forgot that branches aren't always created from develop.

**Comment by eblake003** [comment]
The branch is not based at develop and needs to be remade.

**Pull request closed without merging**